server/plugin.go

```diff
@@ -395,3 +400,30 @@ func generateSecret() (string, error) {
 	s = s[:32]
 	return s, nil
 }
+
+func generateRandomPassword() string {
```

Comment:
What is the reasoning behind using this `generateRandomPassword` instead of a random ID?

Reply:
on using this feature I was getting an error something like this `createUserOrGuest: Login failed because of invalid password.` in my logs. Password requires these validations whereas the `NewId()` method generates a string without any special characters and no surety on the addition of capital and small letters.
![image](https://user-images.githubusercontent.com/100013900/233634661-e7000b4c-2bd8-4d5d-a2a9-dbad4b8155bc.png)